vitals/InternationalizedText.java

```diff
@@ -0,0 +1,42 @@
+package vitals;
+
+import java.util.*;
+
+public class InternationalizedText {
+	
+String language;
+String country;
+
+public InternationalizedText(String language,String country) {
+this.language = language;
+this.country = country;
+}
+
+public String translate(String text) {
+if (language.isEmpty() && country.isEmpty()) {
+	language = new String("en");

```

Comment:
I detect that this code is problematic. According to the [Performance (PERFORMANCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#performance-performance), [Dm: Method invokes inefficient new String(String) constructor (DM_STRING_CTOR)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dm-method-invokes-inefficient-new-string-string-constructor-dm-string-ctor).
Using the java.lang.String(String) constructor wastes memory because the object so constructed will be functionally indistinguishable from the String passed as a parameter.  Just use the argument String directly.